tests/unit/clients/test_http.py

```diff
@@ -245,6 +245,36 @@ def client(mocker, auth_client):
     assert 2 == len(caplog.records)
 
 
+@pytest.mark.asyncio
+async def test_request_with_other_exception(
+        client, mocker, monkeypatch, caplog):
+    """Correct exception is raised when the aiohttp request itself fails."""
+
+    async def mock_refresh_token():
+        pass
+
+    monkeypatch.setattr(
+        client._auth_client, 'refresh_token', mock_refresh_token)
+
+    async def mock_valid_token_set():
```

Comment:
These two mocks could be generalized, since they do the same thing, i.e. `async def mock_coro`.

Reply:
Good point.